include/z64animation.h

```diff
@@ -239,95 +239,95 @@ s16 Animation_GetLength(void* animation);
 s16 Animation_GetLastFrame(void* animation);
 
 /*
- * Animation requests
+ * Animation Task Queue
  */
 
 typedef enum {
-    /* 0 */ ANIMENTRY_LOADFRAME,
-    /* 1 */ ANIMENTRY_COPYALL,
-    /* 2 */ ANIMENTRY_INTERP,
-    /* 3 */ ANIMENTRY_COPYTRUE,
-    /* 4 */ ANIMENTRY_COPYFALSE,
-    /* 5 */ ANIMENTRY_MOVEACTOR
-} AnimationType;
+    /* 0 */ ANIMTASK_LOAD_PLAYER_FRAME,
+    /* 1 */ ANIMTASK_COPY,
+    /* 2 */ ANIMTASK_INTERP,
+    /* 3 */ ANIMTASK_COPY_USING_MAP,
+    /* 4 */ ANIMTASK_COPY_USING_MAP_INVERTED,
+    /* 5 */ ANIMTASK_ACTOR_MOVE
+} AnimTaskType;
 
 typedef struct {
-    /* 0x000 */ DmaRequest req;
-    /* 0x020 */ OSMesgQueue msgQueue;
-    /* 0x038 */ OSMesg msg;
-} AnimEntryLoadFrame; // size = 0x3C
+    /* 0x00 */ DmaRequest req;
+    /* 0x20 */ OSMesgQueue msgQueue;
+    /* 0x38 */ OSMesg msg;
+} AnimTaskLoadPlayerFrame; // size = 0x3C
 
 typedef struct {
-    /* 0x000 */ u8 queueFlag;
-    /* 0x001 */ u8 vecCount;
-    /* 0x004 */ Vec3s* dst;
-    /* 0x008 */ Vec3s* src;
-} AnimEntryCopyAll; // size = 0xC
+    /* 0x00 */ u8 group;
+    /* 0x01 */ u8 vecCount;
+    /* 0x04 */ Vec3s* dest;
+    /* 0x08 */ Vec3s* src;
+} AnimTaskCopy; // size = 0xC
 
 typedef struct {
-    /* 0x000 */ u8 queueFlag;
-    /* 0x001 */ u8 vecCount;
-    /* 0x004 */ Vec3s* base;
-    /* 0x008 */ Vec3s* mod;
-    /* 0x00C */ f32 weight;
-} AnimEntryInterp; // size = 0x10
+    /* 0x00 */ u8 group;
+    /* 0x01 */ u8 vecCount;
+    /* 0x04 */ Vec3s* base;
+    /* 0x08 */ Vec3s* mod;
+    /* 0x0C */ f32 weight;
+} AnimTaskInterp; // size = 0x10
 
 typedef struct {
-    /* 0x000 */ u8 queueFlag;
-    /* 0x001 */ u8 vecCount;
-    /* 0x004 */ Vec3s* dst;
-    /* 0x008 */ Vec3s* src;
-    /* 0x00C */ u8* copyFlag;
-} AnimEntryCopyTrue; // size = 0x10
+    /* 0x00 */ u8 group;
+    /* 0x01 */ u8 vecCount;
+    /* 0x04 */ Vec3s* dest;
+    /* 0x08 */ Vec3s* src;
+    /* 0x0C */ u8* copyMap;
+} AnimTaskCopyUsingMap; // size = 0x10
 
 typedef struct {
-    /* 0x000 */ u8 queueFlag;
-    /* 0x001 */ u8 vecCount;
-    /* 0x004 */ Vec3s* dst;
-    /* 0x008 */ Vec3s* src;
-    /* 0x00C */ u8* copyFlag;
-} AnimEntryCopyFalse; // size = 0x10
+    /* 0x00 */ u8 group;
+    /* 0x01 */ u8 vecCount;
+    /* 0x04 */ Vec3s* dest;
+    /* 0x08 */ Vec3s* src;
+    /* 0x0C */ u8* copyMap;
+} AnimTaskCopyUsingMapInverted; // size = 0x10
```

Comment:
`Inverted` makes me think that this inverts the limb orientations or something. Maybe something like set complement? That could also be confusing though.

Reply:
Yeah Im trying to convey its the inverse/reverse operation of "CopyUsingMap". I dont have any better ideas at the moment.
For what its worth this entry type is never used in practice by any actor, and isnt too important. That ofc does not mean we should neglect giving it a good name. Just putting it out there

Reply:
It seems to be used by player? But yeah it's fine

Reply:
Yeah that block of code is dead because `upperAnimBlendWeight` never gets set to non-zero